src/modules/verification/verification.service.ts

```diff
@@ -0,0 +1,97 @@
+import { BadRequestException, Injectable, Logger } from '@nestjs/common';
+import { ConfigurationService } from '../../configuration/configuration.service';
+
+const Twilio = require('twilio');
+
+@Injectable()
+export class VerificationService {
+  private readonly logger = new Logger(VerificationService.name);
+  private readonly expirationTime = 900;
+
+  private twilioClient;
+  private sendingPhoneNumber: string;
+  private appHash: string;
+
+  // TODO: Replace with Database, Memcache or Redis?
+  private verificationMap = new Map<string, number>();
+
+  constructor(configService: ConfigurationService) {
+    let message = null;
+    if (configService.get('TWILIO_API_KEY') == null ||
```

Comment:
We can leave all the validation for the moment, but should definitely build an env variable validation: 

```suggestion
    // TODO: build env variable validation
    if (configService.get('TWILIO_API_KEY') == null ||
```